lib/ts/recipe/emailpassword/components/library/formBase.tsx

```diff
@@ -240,16 +240,16 @@ export default class FormBase<T> extends PureComponent<FormBaseProps<T>, FormBas
                 }));
                 return;
             }
-
-            // Otherwise if message, set generalError
-            if (result.status === "GENERAL_ERROR") {
+        } catch (e) {
+            if ((e as any).isSuperTokensGeneralError) {
                 this.setState((oldState) => ({
                     ...oldState,
                     status: "GENERAL_ERROR",
-                    generalError: result.message,
+                    generalError: (e as any).message,
```

Comment:
So translation works when rendering the text, if the user provides a text that is not present in the translation keys then it will render as is. If they provide a text that is in the translation keys it will be translated correctly.

Reply:
Okay!